linux_os/guide/system/auditing/package_libaudit_installed/rule.yml

```diff
@@ -1,10 +1,10 @@
 documentation_complete: true
 
-title: 'Ensure the audit-libs package as a part of audit Subsystem is Installed'
+title: 'Ensure the libaudit package as a part of audit Subsystem is Installed'
```

Comment:
It is not a good idea to rename an existing rule:
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#agreements

You should create a new rule if a new name is indeed necessary.
Otherwise, I would recommend to keep the existing name and use Jinja2 in the rule description.

Reply:
Agree on this point :+1: if only the package name is changed it would be less disruptive change and later on every platform can decide utilise it with its own package naming